### ISSUE TITLE
Graph date sometimes returns 1970-01-01T00:00:00Z

### ISSUE THREAD

**Issue by 1papaya** [GIScience/openrouteservice#968]
#### Here's what I did

- Build a `foot-hiking` route on ors.org/maps
- Download ORS JSON

---
#### Here's what I got

The output ORS JSON is huge so I chopped a bit off of it but kept most of the relevant bits for here:
```
{
  "polygons": [],
  "options": {
    "profile": "foot-hiking",
    "preference": "recommended",
    "zoom": 18
  },
  "places": [ ],
  "origin": "response",
  "mode": "directions",
  "isRouteData": true,
  "rawData": {
    "type": "FeatureCollection",
    "features": [ ],
    "bbox": [-17.934016, 28.494079, 3.55, -17.738845, 28.803427, 1391.83],
    "metadata": {
      "attribution": "openrouteservice.org | OpenStreetMap contributors",
      "service": "routing",
      "timestamp": 1621293213608,
      "query": {
        "coordinates": [
          [-17.7735563, 28.8024499],
          [-17.764184474945072, 28.800889904020526],
          [-17.760043144226078, 28.800983920682423],
          [-17.75421738624573, 28.782837133697146],
          [-17.74497985839844, 28.762241877092478],
          [-17.744314670562748, 28.735151517459432],
          [-17.7676711, 28.6896535],
          [-17.7689527, 28.6864356],
          [-17.7677534, 28.6806809],
          [-17.768872976303104, 28.67849304509023],
          [-17.766351699829105, 28.664194192175618],
          [-17.7794634, 28.6614117],
          [-17.881078, 28.65083],
          [-17.9074291, 28.6564579],
          [-17.9107022, 28.6576491],
          [-17.9182375, 28.6597569],
          [-17.920943498611454, 28.659468270016124],
          [-17.9331938, 28.6411776],
          [-17.931468486785892, 28.639432513211013],
          [-17.92955875396729, 28.637718751074104],
          [-17.923936843872074, 28.637379761840798],
          [-17.908401489257816, 28.631616777315863],
          [-17.894668579101566, 28.627134237220695],
          [-17.86033630371094, 28.5274896351413],
          [-17.8445432, 28.4941935]
        ],
        "profile": "foot-hiking",
        "preference": "recommended",
        "format": "geojson",
        "units": "km",
        "language": "en",
        "instructions_format": "html",
        "elevation": true,
        "extra_info": ["surface", "steepness", "waytype"]
      },
      "engine": {
        "version": "6.4.3",
        "build_date": "2021-05-11T10:13:47Z",
        "graph_date": "1970-01-01T00:00:00Z"
      }
    }
  },
  "timestamp": 1623057009516
}
```

As you can see the rawData>engine>graph_date is at 1970 (UNIX Epoch). I have tried to reproduce this but so far each other request I try graph date is a realistic date, but figure the issue is worth it to post.

I figure this could be a bug with ORS and not the maps frontend but could also be wrong about that. Happy to provide more details / the original ORS JSON

**Comment by aoles** [comment]
Is this still happening with recent requests? The issue was supposed to be fixed by https://github.com/GIScience/openrouteservice/pull/927  which has been released 3 weeks ago.

**Comment by 1papaya** [comment]
@aoles The JSON data I posted is just a hair more recent than three weeks (20 days, Tue 18 May 2021 12∶18∶07 AM WEST). However it could be related to the same issue and have been fixed already

**Comment by aoles** [comment]
Thanks @geodavey for the update! The fixed version 6.5.0 was released on May 17th, however, the version on the servers have been updated only a few days later. So I assume your request was still running against the old code. I believe the issue has been already fixed so I will close it for now. If there are still any sort of problems feel free to report them here and we will reopen the issue. Cheers!
